src/core/expr/fexpr.cc

```diff
@@ -433,15 +433,21 @@ DECLARE_METHOD(&PyFExpr::cumsum)
 
 oobj PyFExpr::fillna(const XArgs& args) {
   auto fillnaFn = oobj::import("datatable", "fillna");
-  oobj reverse = args[0]? args[0].to_oobj() : py::obool(false);
-  return fillnaFn.call({this, reverse});
+  oobj value = args[0].to_oobj_or_none();
+  oobj reverse = args[1].to_oobj_or_none();
+
+  py::odict kws;
```

Comment:
@oleksiyskononenko What's the advantage of the keywords arguments here? Learning mode for me

Reply:
You had parameters `value` and `reverse` both positional and keyword. I've changed them to be keyword only, so have to pass them as a dictionary here.

The issue with `value` and `reverse` be positional is that it becomes not obvious what smth like this `dt.fillna(f[:], True)` would mean. It could either be `dt.fillna(f[:], value=True)` or `dt.fillna(f[:], reverse=True)`. Of course, once can look at the order of parameters and realize that it is `value=True` that is passed to the function. However, it is pretty easy to miss that.